Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Action/ActionManager.h

```diff
@@ -32,6 +32,8 @@ namespace AzToolsFramework
             const QList<QAction*>& contextActions, const QList<QAction*>& widgetActions, const QKeySequence& shortcutKeySequence);
 
         EditorActionContext* m_editorActionContext = nullptr;
+        bool m_keyQueued = false;
+        int m_key;
```

Comment:
suggestion: This is the perfect place for an `AZStd::optional` 🙂 (you then don't need the separate `bool` 🥳)

Reply:
Had to switch to another different approach, but I was also thinking of an optional in this case :D